apps/frontend/src/pages/Navbar.tsx

```diff
@@ -78,7 +78,7 @@ function Navbar() {
               >
                 SIGN UP
               </button>
-              <Link to="/login" style={{ textDecoration: 'none' }}></Link>
+              <Link to="/login" style={{ textDecoration: 'none' }}>
```

Comment:
Did you mean to include this change?

Reply:
Oh this is a change from one of my past tickets to fix the login button. Looks like I accidentally branched from that ticket instead of main. The change is correct though, link tag should enclose the login button

Reply:
Alright 👍 In the future, it would be ideal to have separate MR's for changes that serve different purposes